src/main/scala/com/typesafe/sbt/packager/archetypes/JavaServerApplication.scala

```diff
@@ -234,14 +234,14 @@ object JavaServerAppPackaging extends AutoPlugin {
   }
 
   protected def startScriptMapping(name: String, script: Option[File], loader: ServerLoader, scriptDir: String): Seq[LinuxPackageMapping] = {
-    val (path, permissions) = loader match {
-      case Upstart => ("/etc/init/" + name + ".conf", "0644")
-      case SystemV => ("/etc/init.d/" + name, "0755")
-      case Systemd => ("/usr/lib/systemd/system/" + name + ".service", "0644")
+    val (path, permissions, isConf) = loader match {
+      case Upstart => ("/etc/init/" + name + ".conf", "0644", "true")
+      case SystemV => ("/etc/init.d/" + name, "0755", "false")
```

Comment:
I don’t have anything else significant for the moment :)

Version `1.0.0` is looking really great, especially with the auto plugins!